python_plus_gis.py

```diff
@@ -117,3 +117,22 @@ def createPolyGeom(coords):
 # that we just created, and print it out.
 # To make things more reusable: write creation of the LineString and calculating the average 
 #     distance into dedicated functions and use them.
+
+lines = []
+def linestringing (a, b):
+    for p, pp in zip(orig_points, dest_points):        
```

Comment:
give these better names! It's hard to follow the code when they're called `p` and `pp`